openelis.sh

```diff
@@ -0,0 +1,13 @@
+ant dist
```

Comment:
@rajasijaywant3129 what is the purpose of this script?

Reply:
By Mistakenly pushed, I will remove it